src/scene.cpp

```diff
@@ -212,3 +219,27 @@ void Scene::DrawBackground() {
 Graphics::State &Scene::GetGraphicsState() {
 	return state;
 }
+
+
+inline void Scene::DebugValidate(const char* caller) {
+	if (instances.size() <= 1) {
+		//Scene of size 1 happens before graphics stack is up. Which can
+		//cause the following logs to crash.
```

Comment:
minor style-nitpick: no space between // and text

Reply:
Fixed